test/cpp/tensorexpr/test_kernel.cpp

```diff
@@ -372,52 +470,57 @@ void testKernelSumOneAxis() {
 void testKernelSumMultipleAxes() {
   // Test lowering of sum on multiple axes.
   const auto graph_template = R"IR(
-      graph(%0 : Float(5:72,3:24,4:6,6:1, device=cpu)):
+      graph(%0 : Float(2:18,3:6,2:3,3:1, device=cpu)):
```

Comment:
Wasn't that a part of #43905?

Reply:
Yes, see the comment - once I land that PR I'll rebase and it'll go away.